gap/Bialgebroids.gi

```diff
@@ -833,6 +833,29 @@ InstallMethod( AddAntipode,
     
 end );
 
+##
+InstallMethod( IsCommutative,
+        "for an Algebroid",
+        [ IsAlgebroid ],
+     
+  function( A )
+    local arrows, is_commutative, x, y;
+    
+    arrows := Arrows( UnderlyingQuiverAlgebra( A ) );
+    
+    is_commutative := true;
+
+    for x in arrows do
+      for y in arrows do
```

Comment:
This code is far from optimal. You are checking the whole square, whereas you only need to check the upper/lower triangle (excluding the diagonal).